src/jsdocUtils.js

```diff
@@ -126,17 +126,45 @@ const hasDefinedTypeReturnTag = (tag) => {
   return true;
 };
 
+const tagsWithOptionalType = [
```

Comment:
I would prefer that this variable would not include those items within `tagsWithMandatoryType` and that `tagMightHaveType` would instead check within both `tagsWithMandatoryType` and `tagsWithOptionalType`.

Reply:
Makes sense, done